### ISSUE TITLE
Support compsitionUpdatePolicy default value

### ISSUE THREAD

**Issue by bobh66** [crossplane/crossplane#3745]
### What problem are you facing?
The default value for compositionUpdatePolicy is Automatic which preserves the existing behavior. In order to fully take advantage of CompositionRevisions the compositionUpdatePolicy must be explicitly set to Manual on every Claim.

### How could Crossplane help solve your problem?
It would be very useful to be able to set a defaultCompositionUpdatePolicy on the XRD and have it copied to the Claim/Composite if it is not overridden in the Claim.  That would allow for composites to have a Manual composition update policy by default.

**Comment by negz** [comment]
This sounds reasonable to me. Is this something you'd be interested in working on? If so, should we add it to the v1.12 milestone?

**Comment by bobh66** [comment]
Yes - I think it's close enough to #3744 that I can handle both, though the implementation will likely be different.

**Comment by phisco** [comment]
was addressed by #3963
